packages/storybook/.storybook/head.html

```diff
@@ -0,0 +1 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"></link>
```

Comment:
this is a way to add a font to storybook so that all elements use that font, so that it is standardized, without having to specify it on each element.